src/main/kotlin/org/jitsi/nlj/rtp/codec/vp8/Vp8Packet.kt

```diff
@@ -24,10 +24,36 @@ import org.jitsi.rtp.extensions.bytearray.hashCodeOfSegment
 import org.jitsi_modified.impl.neomedia.codec.video.vp8.DePacketizer
 
 class Vp8Packet(
-    data: ByteArray,
+    buffer: ByteArray,
     offset: Int,
     length: Int
-) : VideoRtpPacket(data, offset, length) {
+) : VideoRtpPacket(buffer, offset, length) {
+    override val isKeyframe: Boolean by lazy(LazyThreadSafetyMode.PUBLICATION) {
```

Comment:
did you use `LazyThreadSafetyMode.PUBLICATION` because you think the cost of redundantly calculating the value would be less than synchronizing to wait on the initial result?

Reply:
Yeah, basically.  It's not like it matters that much either way.  Do you think it'd be cleaner to leave it out just to be less noisy?